cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -275,4 +278,20 @@ public static void parsePodTemplate(AbstractModel model, PodTemplate pod)   {
             model.templateSecurityContext = pod.getSecurityContext();
         }
     }
+
+    /**
+     * If the provided Storage instance is a persistent claim one or a jbod containing at least a persistent volume
+     *
+     * @param storage the Storage instance to check
+     * @return
```

Comment:
Either include `@return` with a sentence of doc, or omit it on the basis that the method doc itself is documenting the return value.